docs/Policy/Maintenance/Scenario_Maintenance_Policy.md

```diff
@@ -0,0 +1,189 @@
+
+ For example if we had
+ ```
+tests:
+- as: acm-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+      SUB_CHANNEL: release-2.5
+      SUB_INSTALL_NAMESPACE: open-cluster-management
+      SUB_PACKAGE: advanced-cluster-management
+      SUB_SOURCE: redhat-operators
+      SUB_TARGET_NAMESPACES: open-cluster-management
+    test:
+    - chain: interop-acm
+    workflow: ipi-aws
+- as: acm-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+      SUB_CHANNEL: release-2.6
+      SUB_INSTALL_NAMESPACE: open-cluster-management
+      SUB_PACKAGE: advanced-cluster-management
+      SUB_SOURCE: redhat-operators
+      SUB_TARGET_NAMESPACES: open-cluster-management
+    test:
+    - chain: interop-acm
+    workflow: ipi-aws
+```
+And we wanted to deprecate testing `ACM2.5` then we will just remove the following test stanza
+```
+- as: acm-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+      SUB_CHANNEL: release-2.5
+      SUB_INSTALL_NAMESPACE: open-cluster-management
+      SUB_PACKAGE: advanced-cluster-management
+      SUB_SOURCE: redhat-operators
+      SUB_TARGET_NAMESPACES: open-cluster-management
+    test:
+    - chain: interop-acm
+    workflow: ipi-aws
+```
+While scenario expansions must have a passing rehearsal job for the PR to be merged a deprecation does not.
+
+## OpenShift Deployment
+>This responsibility is abstracted away from both the CSPI-QE team and PQE teams to the workflow OWNERS. 
+
+This responsibility falls to whoever the own of the workflow that is being used to deploy the OpenShift environment needed by the OpenShift Layered Product Scenario. Each workflow will most likely have a different owner. If you need support for a problem in a workflow you'll need to find the workflow that you are using in the [release repo's step-registry](https://github.com/openshift/release/tree/master/ci-operator/step-registry) and contact the users in the OWNERS file.
+
+## Layered Product Deployment
+>This is the responsibility of the PQE team.
+
+The model being followed by this [layered product onboarding](../../Onboarding/Onboarding_Guide.md) is built on the idea that automation that is already created and being maintained should not be recreated by a different team attempting to achieve the same result. Therefore the onboarding of the scenario will ensure that the layered product deployment relies on automation built by that products QE team. If there are failures in the OCP CI scenario for the layered product deployment then it will need to be fixed at the source, which will be the layered product QE team's repositories.
```

Comment:
```suggestion
The model being followed by this [layered product onboarding](../../Onboarding/Onboarding_Guide.md) is built on the idea that automation that is already created and being maintained should not be recreated by a different team attempting to achieve the same result. Therefore the onboarding of the scenario will ensure that the layered product deployment relies on automation built by that product's QE team. If there are failures in the OCP CI scenario for the layered product deployment then it will need to be fixed at the source, which will be the layered product QE team's repositories.
```